code/renderers/react/package.json

```diff
@@ -31,6 +31,11 @@
       "require": "./dist/config.js",
       "import": "./dist/config.mjs"
     },
+    "./testing-api": {
```

Comment:
maybe we call it test-utils, like vue does with `@vue/test-utils`?
come to think of it, we'll be using `composeStories` for other 'portable stories' scenarios, right? not just testing. It's actually the only way to provide users a way to [reuse stories within stories](https://storybook.js.org/docs/react/writing-stories/stories-for-multiple-components) using CSF3, for instance.

Reply:
Agree with it 👍 . Seems like a reasonable choice for what the feature aims to accomplish

Reply:
Updated per feedback but I think we can leave this until we actually need it. When we delete SSv6 in v8 we won't actually need it.